### PR TITLE
Fix Failing E2E Tests

### PR DESCRIPTION
**Pull request by Convly**
### What does it do?

- fix several bugs causing tests to fail
- fix several broken tests
- skip some i18n tests that are failing from a bug; will re-enable when bug is fixed

### Why is it needed?

Too many tests were failing, and without the e2e test requirement on in the CI more were being added. This gets us back to a state where we can rely on e2e tests again 🎉

### How to test it?

All tests should pass.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx [hunk 1]
```diff
@@ -770,19 +770,22 @@ const UpdateAction: DocumentActionComponent = ({
...
-            message: formatMessage({
-              id: 'content-manager.validation.error',
-              defaultMessage:
-                'There are validation errors in your document. Please fix them before saving.',
-            }),
-          });
-
-          return;
+        // Don't run the validation for drafts
+        if (activeTab !== 'draft') {
```
Comment (Convly) [inline packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx:774]: This is not what we should do. We should just run some validation and skip others instead.

File: packages/core/content-manager/admin/src/utils/validation.ts [hunk 2]
```diff
@@ -287,7 +302,7 @@ const addMinValidation: ValidationFn =
         (attribute.type === 'component' && attribute.repeatable) ||
         attribute.type === 'dynamiczone'
       ) {
-        if (!attribute.required && 'test' in schema && min) {
+        if (options.status !== 'draft' && !attribute.required && 'test' in schema && min) {
```
Comment (Convly) [inline packages/core/content-manager/admin/src/utils/validation.ts:305]: Should we also handle https://github.com/strapi/strapi/pull/20691/files#diff-c186be7a068bde5fa18b001d2973bfe8a3981eb96e405ef4b87d0a80be5a7dedR330 the same way?

Comment (innerdvations) [reply packages/core/content-manager/admin/src/utils/validation.ts:305]: Yeah, but let's handle it in another PR, we need to get this merged ASAP to get tests fixed

**Review by Bassel17 — APPROVED** [review]
LGTM !

**Review by remidej — APPROVED** [review]

**Pull request merged**